src/pages/developers/tutorials.tsx

```diff
@@ -43,43 +39,50 @@ import {
 import externalTutorials from "@/data/externalTutorials.json"
 
 import { useBreakpointValue } from "@/hooks/useBreakpointValue"
-import { useRtlFlip } from "@/hooks/useRtlFlip"
-
-const FilterTag = forwardRef<{ isActive: boolean; name: string }, "button">(
```

Comment:
Hey sorry for not noticing this before, but we are doing the tag migration in a different PR #13976 and since the visual results in this PR for it are not perfect, I think it will be just easier to revert this change.
![image](https://github.com/user-attachments/assets/09755382-9071-474b-b16d-43b9b16af732)

vs prod
![image](https://github.com/user-attachments/assets/5a8a8fb0-3c38-4b79-b622-7ddbdb1b500d)